src/server/project.ts

```diff
@@ -1730,6 +1731,17 @@ namespace ts.server {
                 this.projectService.openFiles,
                 (_, fileName) => this.projectService.tryGetDefaultProjectForFile(toNormalizedPath(fileName)) === this);
         }
+
+        /*@internal*/
+        watchNodeModulesDirectory(directoryPath: string, cb: DirectoryWatcherCallback) {
+            return this.projectService.watchFactory.watchDirectory(
+                directoryPath,
+                cb,
+                WatchDirectoryFlags.Recursive,
+                this.projectService.getWatchOptions(this),
+                WatchType.NodeModulesForClosedScriptInfo

```

Comment:
incorrect watch type?

Reply:
Yeah, this was just a mistake, but if I do want to share the watches with the closed script info watches (and I think I do—isn’t sharing minimizing recursive directory watches by sharing them good practice?), I guess I will end up using this type?